src/tests/unit/tests/popup/incompatible-browser-renderer.test.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import ReactDOM from 'react-dom';
+import { It, Mock } from 'typemoq';
+
+import { IncompatibleBrowserRenderer } from '../../../../popup/incompatible-browser-renderer';
+
+describe('IncompatibleBrowserRenderer', () => {
+    it('render', () => {
+        const renderMock = Mock.ofType<typeof ReactDOM.render>();
+        const containerStub = Mock.ofType<HTMLElement>();
+        const documentMock = Mock.ofType<NodeSelector & Node>();
+
+        documentMock.setup(mock => mock.querySelector('#popup-container')).returns(() => containerStub.object);
+
+        renderMock
+            .setup(mock => mock(It.isAny(), containerStub.object))
```

Comment:
should the first parameter be `containerStub`?

Reply:
I think this is correct - this is mocking [ReactDOM.render](https://reactjs.org/docs/react-dom.html#render) where the first parameter is the element and the second parameter is the container.

Reply:
Nope! The first is the react component. The 2nd is the container to which it will be added.

Reply:
Right! I'm reading backwards. My bad.